crates/sui-config/src/node.rs

```diff
@@ -152,6 +152,7 @@ impl NodeConfig {
 pub struct ConsensusConfig {
     pub consensus_address: Multiaddr,
     pub consensus_db_path: PathBuf,
+    pub delay_step: Option<u64>,
```

Comment:
Alternatively I think it may be cleaner if you don't use `Option` here but provide a serde default?

Reply:
Oh, too late :)
I don't really have a strong opinion on that, can use default next time

Reply:
The benefit of serde default is that we can then deploy this without a new config file

Reply:
I think it is same with Option, it deserializes to None by default